packages/mgt-components/src/utils/SvgHelper.ts

```diff
@@ -266,20 +286,41 @@ export function getSvg(svgIcon: SvgIcon, color?: string) {
         </svg>
       `;
 
+    case SvgIcon.SmallEmailHovered:
+      return html`
+      <svg width="16" height="13" viewBox="0 0 16 13" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
these new hover state svgs need to have the same width and height as the unhovered icon otherwise there are layout shifts as you move though hover states